components/Avtar.css

```diff
@@ -0,0 +1,209 @@
+@import url('https://fonts.googleapis.com/css2?family=Merriweather:ital,wght@1,900&family=Oooh+Baby&family=Open+Sans:wght@300&display=swap');
+/* css for navigation */
+
+:root{
```

Comment:
Use :root only in main css file and try to avoid use of I'd for css selector, you may need those styles in multiple places.

Reply:
I have made CSS file different-different that's why I have used :root in Avatar.